src/azure-cli/azure/cli/command_modules/acs/custom.py

```diff
@@ -2084,6 +2084,25 @@ def aks_create(cmd, client, resource_group_name, name, ssh_key_value,  # pylint:
                no_wait=False,
                yes=False,
                enable_azure_rbac=False):
+    # decorator pattern
+    raw_parameters = locals()
```

Comment:
separate this into a different PR, so we can revert this easily.

Reply:
Sure, changed in commit [#93a006f](https://github.com/Azure/azure-cli/pull/19583/commits/93a006f79172e8f3c5338e4b3cd05502f4cd361d).